server/server.js

```diff
@@ -136,11 +145,28 @@ var upload = multer({
  */
 var exonFile = bindingPath + 'exons.bin';
 
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
 /**
- * POST request is not used as it conflicts with jsonp.
+ * User log POST handler.
+ */
+app.post('/genotet/log', function(req, res) {
+  log.serverLog('POST', req.body.type);
+
+  var prefix = userPath;
+  switch (req.body.type) {
+    case 'user-log':
```

Comment:
If the request is posted to dedicated url like '/genotet/log', then it might not be necessary to further include req.body.type, because anything posted to this url must be user-log.